src/styles/blocks/main.scss

```diff
@@ -0,0 +1,26 @@
+
+.main {
+  margin: 0 auto;
+
+  &__catalog {
+    width: $catalog-width;
+    padding: 50px 0;
+    margin: 0 auto;
+    box-sizing: border-box;
+  }
+
+  &__list {
+    display: flex;
+    flex-wrap: wrap;
+
+    &-item:not(:nth-child(4n)) {
+      box-sizing: border-box;
+      margin-right: $space-column;
+      margin-bottom: $space-row;
+    }
```

Comment:
Using this selector box-sizing is not set to every 4th element, so I think it could cause an unexpected behavior. Also I'm sure that 4th element in the first row should have margin-bottom as well